syntax/std-grammar.md

```diff
@@ -0,0 +1,7 @@
+# grammar
+
+`arrai` features the ability to parse grammar and use that grammar to parse
+inputs as part of its standard library. This package provides that functionalities
+and built-in grammars to use.
```

Comment:
```suggestion
inputs as part of its standard library. This package provides that functionality
and supplies built-in grammars to use.
```

Reply:
Maybe some indication of what the built in grammars are?

Reply:
@marcelocantos will be writing this part of the docs, I'm not very familiar either.